samples/Chat/src/app/utils/useIsMobile.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
previously didn't exist in the chat sample, this is a direct copy that call sample and callwithchat samples have